pkg/reconciler/cronjobsource/resources/eventtype.go

```diff
@@ -39,7 +39,7 @@ func MakeEventType(src *v1alpha1.CronJobSource) *eventingv1alpha1.EventType {
 		Spec: eventingv1alpha1.EventTypeSpec{
 			Type:   v1alpha1.CronJobEventType,
 			Source: v1alpha1.CronJobEventSource(src.Namespace, src.Name),
-			Broker: src.Spec.Sink.Name,
+			Broker: src.Spec.Sink.Ref.Name,
```

Comment:
Does this support the deprecated version?

Reply:
Fixed. Thanks

Reply:
does now `args.Src.Spec.Sink.GetRef().Name,`